tailwind.config.js

```diff
@@ -0,0 +1,15 @@
+/** @type {import('tailwindcss').Config} */
+export default {
```

Comment:
can you coordinate this with @jamesBariuad , if this works with his changes. Seems like it might just clash